### PR TITLE
Finish Advanced login

### PR DESCRIPTION
**Pull request by tranxuanhiep**
image

### CODE REVIEW

**Comment by tranxuanhiep** [comment]
ready

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -3,15 +3,36 @@ def log_in user
...
+    end
   end
 
   def logged_in?
     current_user.present?
   end
 
+  def forget user
+    user.forget
+    cookies.delete(:user_id)
```
Comment (hhhhuy112) [inline app/helpers/sessions_helper.rb]: remove ()

File: app/helpers/sessions_helper.rb [hunk 2]
```diff
@@ -3,15 +3,36 @@ def log_in user
...
+    cookies.delete(:remember_token)
```
Comment (hhhhuy112) [inline app/helpers/sessions_helper.rb]: remove()

File: app/helpers/sessions_helper.rb [hunk 3]
```diff
@@ -3,15 +3,36 @@ def log_in user
...
+  end
+
   def log_out
+    forget(current_user)
```
Comment (hhhhuy112) [inline app/helpers/sessions_helper.rb]: remove ()

File: app/helpers/sessions_helper.rb [hunk 4]
```diff
@@ -3,15 +3,36 @@ def log_in user
...
     session.delete(:user_id)
```
Comment (hhhhuy112) [inline app/helpers/sessions_helper.rb]: remove()

File: app/views/sessions/new.html.erb [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <% provide :title, t(".title") %>
-<h1>Log in</h1>
+<h1><%= t(".title") %></h1>
```
Comment (hhhhuy112) [inline app/views/sessions/new.html.erb]: remove ()
 t ".title"

File: app/views/sessions/new.html.erb [hunk 6]
```diff
@@ -11,9 +11,14 @@
       <%= f.label :password %>
       <%= f.password_field :password, class: "form-control" %>
 
+      <%= f.label :remember_me, class: "checkbox inline" do %>
+        <%= f.check_box :remember_me %>
+        <span><%= t(".content1") %></span>
```
Comment (hhhhuy112) [inline app/views/sessions/new.html.erb]: t ".content1"

File: app/views/sessions/new.html.erb [hunk 7]
```diff
@@ -11,9 +11,14 @@
...
+      <% end %>
+
       <%= f.submit t(".title"), class: "btn btn-primary" %>
     <% end %>
 
-    <p>New user? <%= link_to t(".content0"), signup_path %></p>
+    <p><%= t(".content2") %> <%= link_to t(".content0"), signup_path %></p>
```
Comment (hhhhuy112) [inline app/views/sessions/new.html.erb]: t ".content2"

File: app/views/static_pages/about.html.erb [hunk 8]
```diff
@@ -1,7 +1,7 @@
 <% provide :title,(t ".title")  %>
 <h1><%= t ".title"%></h1>
 <p>
-  <%= t ".content0"%> <a href="http://www.railstutorial.org/"><em><%= t "static.home.content1"%></em></a> is a
+  <%= t ".content0"%> <a href="http://www.railstutorial.org/"><em><%= t "static_pages.home.content1"%></em></a> is a
```
Comment (hhhhuy112) [inline app/views/static_pages/about.html.erb]: space <%= t ".content0" %> <%= t "static_pages.home.content1" %>

File: app/views/static_pages/help.html.erb [hunk 9]
```diff
@@ -4,6 +4,6 @@
   <%= t ".content0"%>
   <a href="http://www.railstutorial.org/help"><%= t ".content1"%></a>.
   <%= t ".content2"%>
-  <a href="http://www.railstutorial.org/book"><em><%= t "static.home.content1"%>l</em>
+  <a href="http://www.railstutorial.org/book"><em><%= t "static_pages.home.content1"%>l</em>
```
Comment (hhhhuy112) [inline app/views/static_pages/help.html.erb]: <%= t "static_pages.home.content1" %>

File: app/views/static_pages/help.html.erb [hunk 10]
```diff
@@ -4,6 +4,6 @@
...
   <%= t ".content3"%></a>
```
Comment (hhhhuy112) [inline app/views/static_pages/help.html.erb]: <%= t ".content3" %>

**Comment by hhhhuy112** [comment]
commented

**Comment by tranxuanhiep** [comment]
ready

**Comment by dothidiemthao** [comment]
merged

**Pull request merged**